scripts/index.js

```diff
@@ -0,0 +1,17 @@
+import pubSub from './pubSub';
+import { generateHaiku } from './HaikuGenerator';
+import Form from './FormComponent';
+import Result from './ResultComponent';
+import Modal from './ModalComponent';
+import Field from './FieldComponent';
+import Screen from './ScreenComponent';
+
+Array.prototype.random = function() {
```

Comment:
Extending the prototype like this is generally something that's frowned upon in JavaScript. With ES6 modules, it's easy to figure out where a function is defined (if it's imported from another module, this is very clear). With something like this, it's much harder to track down where this is coming from. 

Also, if you depend on a library (like underscore), and that library decides it also wants to add a `random` prototype to the Array, you're going to have problems. 

Since you're only using `[].random()` in `HaikuGenerator`, I'd argue that you should just add a `function random(array) { }` function there.